### PR TITLE
Add methods isSunday(), isMonday(), isTuesday() ..

### PR DESCRIPTION
**Pull request by wangchj**

### CODE REVIEW

**Comment by briannesbitt** [comment]
Triple "===" !  Tests?

File: src/Carbon/Carbon.php [hunk 1]
```diff
@@ -1278,6 +1278,69 @@ public function isSameDay(Carbon $dt)
         return $this->toDateString() === $dt->toDateString();
     }
 
+    /**
+     * Checks if this day is a Sunday.
+     * @return boolean
```
Comment (GrahamCampbell) [inline src/Carbon/Carbon.php]: boolean is wrong - bool is correct

**Comment by GrahamCampbell** [comment]
Yeh, we need `===` here.

**Comment by wangchj** [comment]
See https://github.com/briannesbitt/Carbon/pull/395

**Comment by dereuromark** [comment]
You don't need to open new PRs for each commit (change). Just adding the commits on top of your branch automatically updates the current PR.

**Pull request closed without merging**